validation/utils.go

```diff
@@ -0,0 +1,27 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package validation
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ValidateTokenAndServerURL(token string, serverURL string) {
+	if strings.TrimSpace(token) == "" {
+		fmt.Println("Error: Please provide your access token by using the -t/--access-token flag, or by setting the REANA_ACCESS_TOKEN environment variable.")
+		os.Exit(1)
+	}
+
+	if strings.TrimSpace(serverURL) == "" {
+		fmt.Println("Please set REANA_SERVER_URL environment variable.")
```

Comment:
The first validation failure message has leading "Error:", the second validation failure message does not.  Can we harmonise?  

(BTW in the future we may want to introduce some nice info/debug/error logging library with colouring and whatnot.)